cleverhans_tutorials/mnist_tutorial_eager.py

```diff
@@ -0,0 +1,255 @@
+"""
+This tutorial shows how to generate adversarial examples using FGSM
+and train a model using adversarial training with TensorFlow Eager.
+
+It is similar to mnist_tutorial_tf.py.
+mnist_tutorial_tf - dynaminc eager execution.
+mnist_tutorial_tf - graph based execution.
+The original paper can be found at:
+https://arxiv.org/abs/1412.6572
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import logging
+import sys
+import os
+
+from cleverhans.utils_mnist import data_mnist
+from cleverhans.utils import AccuracyReport, set_log_level
+from distutils.version import LooseVersion
+
+import tensorflow as tf
+from tensorflow.python.platform import flags
+from cleverhans.utils_eager import model_train, model_eval
+from cleverhans.attacks_eager import *
+from cleverhans_tutorials.tutorial_models_eager import make_basic_cnn
+
+try:
+    tf.enable_eager_execution()
+except AttributeError:
+    if LooseVersion(tf.__version__) < LooseVersion('1.8.0'):
+        error = ('For eager execution',
+                 'use Tensorflow version greather than 1.8.0.')
+        print(error)
+        exit(1)
+
+if tf.executing_eagerly() is True:
+    print('TF Eager Activated.')
+else:
+    error = 'Error Enabling Eager Execution.'
+    raise Exception(error)
+tfe = tf.contrib.eager
+
+FLAGS = flags.FLAGS
+
+
+# Keeps track of implemented attacks.
+# Maps attack string taken bash to attack class
+# -- 'fgsm' : FastGradientMethod
+
+ListOfAttacks = {
+                'fgsm': FastGradientMethod
```

Comment:
This doesn't seem to follow PEP8 alignment, or does it?

Reply:
Looks misaligned, Not sure, it did pass the travis test though